dojo/tools/veracode/parser.py

```diff
@@ -233,9 +233,9 @@ def __xml_sca_flaw_to_finding(cls, test, report_date, vendor, library, version,
 
         _description = 'This library has known vulnerabilities.\n'
         _description += \
-                "**CVE: [{0}](https://nvd.nist.gov/vuln/detail/{0})** ({1})\n" \
+                "**CVE: [{0}] https://nvd.nist.gov/vuln/detail/{0} ** ({1})\n" \
```

Comment:
The idea here was, to render the CVE as a link in markdown notation. This is not necessary because the CVE has a link in the attribute _Vulnerability Id_. We should change the line to:

```suggestion
                "**CVE:** {0} ({1})\n" \
```

Reply:
Done @StefanFl